templates/webhook.bin.erb

```diff
@@ -30,15 +30,16 @@ else
   raise "Configuration file: #{WEBHOOK_CONFIG} does not exist"
 end
 
-ENV['HOME'] = '/var/lib/peadmin'
-ENV['PATH'] = '/sbin:/usr/sbin:/bin:/usr/bin:/opt/puppet/bin'
+ENV['HOME'] = '/var/lib/<%= @user %>'
+ENV['PATH'] = '/sbin:/usr/sbin:/bin:/usr/bin:<%= if @is_pe == true or @is_pe == 'true' then '/opt/puppet/bin' else '/usr/local/bin' end %>'
 
 $logger = WEBrick::Log::new($config['access_logfile'], WEBrick::Log::DEBUG)
 
+<% if @is_pe == true or @is_pe == 'true' %>
 # PE 3.7 check for renamed certificate path
-public_key_path  = File.join("#{$config['certpath']}", "#{$config['certname']}-cert.pem")
-
-private_key_path = File.join("#{$config['certpath']}", "#{$config['certname']}-private.pem")
+config['public_key_path']  = File.join("#{$config['certpath']}", "#{$config['certname']}-cert.pem")
```

Comment:
This actually needs to be `$config` as its a instance variable above not a local